pkg/utils/k8s/sync/backup.go

```diff
@@ -0,0 +1,17 @@
+package sync
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+
+	extensionv1 "github.com/universityofadelaide/shepherd-operator/pkg/apis/extension/v1"
+)
+
+// Backup function to ensure the Object is in sync.
```

Comment:
Where are these used?

Reply:
In the sync_controller when it's calling `CreateOrUpdate` on the backup